source/com.google.android.gms/play-services-auth-blockstore/Transforms/Metadata.Namespaces.xml

```diff
@@ -0,0 +1,8 @@
+﻿<metadata>
+    <attr 
+        path="/api/package[@name='com.google.android.gms.auth.blockstore']" 
+        name="managedName"
+        >
+        Android.Gms.Auth.Managed.BlockStore
```

Comment:
Although I prefer this capitalization, it looks like Google doesn't:

```
#### New Type: Android.Gms.Auth.Managed.BlockStore.Blockstore
#### New Type: Android.Gms.Auth.Managed.BlockStore.BlockstoreClient
#### New Type: Android.Gms.Auth.Managed.BlockStore.BlockstoreClientConsts
```

We should probably use `Blockstore` so that it matches the capitalization of the type names.